content/docs/sidebar.json

```diff
@@ -661,6 +666,11 @@
             "source": "describe.md",
             "label": "describe"
           },
+          {
+            "slug": "doctor",
+            "source": "doctor.md",
+            "label": "doctor"
```

Comment:
The cmd ref changes are completely unrelated, right? Is the generator script running automatically whenever you change anything? Otherwise 🙏🏼 let's make separate PRs for ease of review in the future. Thanks

Reply:
Yes, they are unrelated. It doesn't run automatically, I do that manually. But, I can do that in separate PRs in future, sure.

Reply:
Thanks! Actually this PR was mostly about that now that I see haha. So I changed the title 😬